Pygame-Buttons/requirements.txt

```diff
@@ -0,0 +1,4 @@
+  | Requirement |  Command         | Documentation |
+--|-------------|------------------|---------------|
+1.| Pygame      |pip install pygame|  Pygame/PyPi  |
+--|-------------|------------------|---------------|
```

Comment:
This not the way of defining the requirements file. See this requirement file: https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Amazon-Price-Alert/requirements.txt

Reply:
Sorry , i didn't knew that